### ISSUE TITLE
Edge of Nowhere Update (Don't work now)

### ISSUE THREAD

**Issue by Javierusan** [LibreVR/Revive#809]
Hi. since I put the update of Edge of Nowhere, it does not work with HTC Vive, there is an error of "You have a pending hardware issue that is preventing you from ..." and don't show the game in VR (only desktop). Any solution please? Thx.

**Comment by unkellsam** [comment]
+1 same here.

**Comment by Javierusan** [comment]
I think that the problem is on Revive that does not support the update of the game. The update was released on November 2.

**Comment by Armada651** [comment]
The problem is that they've updated to a newer version of the Oculus SDK. Can you try the Revive Beta?

**Comment by unkellsam** [comment]
I'll try it and report back.

**Comment by Javierusan** [comment]
I tried with Second Beta Version 1.2.0, and nothing, don't work. Too tried patch the exe and inject, buy don't work :(

**Comment by Armada651** [comment]
It might require the very latest version of the Oculus SDK. You'll have to wait for the next update of Revive.

**Comment by unkellsam** [comment]
Yeah, still doesn't work with the beta. It does not show the same "hardware issue" message through, it just gets stuck on the logo screen on the desktop, and does not open in the headset.

**Comment by Javierusan** [comment]
When will you upload the next release of revive (with support to Edge of Nowhere)? Thank you.

**Comment by Armada651** [comment]
Before the end of the year. I'm still working on it, so it's not as simple as pressing "upload".

**Comment by ytdlder** [comment]
Just wanted to confirm; not working on Revive 1.2.0 beta 2
"...pending hardware issue..."

Thanks in advance! :-)

**Comment by Armada651** [comment]
@ytdlder Beta 3 is already out and should fix it: https://github.com/LibreVR/Revive/releases/tag/1.2.0-beta3

**Comment by ytdlder** [comment]
Ups... as I just updated to beta2 I really didn't check beforehand, sry o_O
I'll try that asap, thanks!

**Comment by ytdlder** [comment]
Ok, just tried with beta3: ist launches, but only to a black screen.
In SteamVR it says Edge of Nowhere is running fine, so that's not the problem. I also tried "Inject", but that didn't help.

Am I missing something? (don't know as I haven't played it yet)

**Comment by bilago** [comment]
working fine on beta 3 for me. First play through, no black screen issue.

**Comment by ytdlder** [comment]
You're launching via Steam-Overlay Revive page, right? No injection / patching? Do you run anything "as Administrator"?

**Comment by bilago** [comment]
Running straight from within Steam-VR overlay, not touching injection or running as administrator

**Comment by ytdlder** [comment]
thx for posting back! Do you have Steam beta enabled by any chance?

**Comment by bilago** [comment]
I'm not enrolled in the steamvr beta

**Comment by ytdlder** [comment]
Ok, I don't know how, but I did get it to work (tried running steam, revive, oculus, EoN as administrator and some other stuff).
Anyhow: is there any way to force supersampling? The resolution is abysmal to say the least!

- SteamVR SS settings don't work via Oculus/Revive
- Oculus Debug-Settings don't seem to work either (https://www.roadtovr.com/improve-oculus-rift-game-image-quality-using-this-tool-oculus-debug-tool/)
- "Advanced Settings": the override setting doesn't work too - at least in this game

cheers!

**Comment by unkellsam** [comment]
We still can't get this to work on the latest revive. The title screen window comes up on the desktop but nothing in the headset. Any updated instructions?

**Comment by unkellsam** [comment]
Any updates on this? 8 months later and I still can't get Edge of Nowhere to work.
